src/domain/consents/{ID}.ts

```diff
@@ -0,0 +1,88 @@
+/*****
```

Comment:
`Rename file from {ID} -> consents`

Add a `index.ts` and re-export all public functions/interfaces within this file.